src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -1561,6 +1615,12 @@ private IEnumerable<ODataProperty> CreateStructuralPropertyBag(SelectExpandNode
 
                 foreach (IEdmStructuralProperty structuralProperty in structuralProperties)
                 {
+                    if (structuralProperty.Type != null && structuralProperty.Type.IsStream())
+                    {
+                        // skip the stream property, the stream property is written in its own logic
+                        continue;
+                    }
```

Comment:
@mikepizzo It's a little bit confusing.

For the stream property serializer, what 's the correct way to do?

1) Create a normal "ODataProperty" and build its value as "ODataStreamReferenceValue"   or
2) Crete a "ODataStreamPropertyInfo" and call "WriteStart(),  WriteEnd()" 

It's "ODataStreamPropertyInfo" only for $expand=photo ?